test/object-safe-get/index.js

```diff
@@ -57,29 +69,74 @@ test('returns undefined for non-existing properties, using array arg', function(
   t.end();
 });
 
+test('returns defaultValue (third parameter) for non-existing properties, using array arg', function(t) {
+  t.plan(8);
+  var obj = {a: {aa: {aaa: 2}}, b: 4, c: null, d: 0};
+  t.ok(compare(get(obj, ['b', 'bb'], 888), 888));
+  t.ok(compare(get(obj, ['a', 'bb'], 888), 888));
+  t.ok(compare(get(obj, ['b', 'bb', 'bbb'], 888), 888));
+  t.ok(compare(get(obj.b, ['bb', 'bbb'], 888), 888));
+  t.ok(compare(get(obj, ['c', 'cc'], 888), 888));
+  t.ok(compare(get(obj, ['d', 'dd', 'ddd'], 888), 888));
+  var arr = ['b', 'bb', 'bbb'];
+  t.ok(compare(get(obj, arr, 888), 888));
+  t.ok(compare(arr, ['b', 'bb', 'bbb'])); // array arg preserved
+
+  t.end();
+});
+
 test('returns undefined for falsey property names using dot notation', function(t) {
-  t.plan(4);
+  t.plan(5);
   var obj = {a: {aa: {aaa: 2}}, b: {}};
-  t.ok(compare(get(obj, 'a, ""', undefined)));
-  t.ok(compare(get(obj, 'a, aa, aaa, ""', undefined)));
-  t.ok(compare(get(obj, 'b, ""', undefined)));
-  t.ok(compare(get(obj, 'b, ""'), undefined));
```

Comment:
those were added here: https://github.com/angus-c/just/commit/67496a797490ff15d2e78fe8de106d6a28188ac1 but I don't understand this - parsing in the lib is done only on `.` and not commas and spaces, so all those above just try to get a one long property key like `'a, aa, aaa, ""'`

Reply:
Yes this was a silly mistake of mine. Thanks for identifying and fixing.